pages/boards/index.tsx

```diff
@@ -0,0 +1,40 @@
+import BestArticleList from "@/components/boards/BestArticleList";
+import AllArticleList from "@/components/boards/AllArticleList";
+import styles from "@/components/boards/Freeboard.module.scss";
+import { ArticleProps } from "@/types/articleTypes";
+import { getArticle } from "@/lib/articleApi";
+
+const ORDERBY = "recent";
+const PAGESIZE = 10;
```

Comment:
상수 부분은 컴포넌트 밖에서 선언해 주신 부분 좋습니다. 👍

Reply:
상수임을 나타내기 위해 대문자로 작성해 주신 부분도 좋습니다. 다만, 보통 네이밍을 할 때 `_`를 사용하여 `scream snake case` 방식으로 작성하는 부분도 참고하세요!
```suggestion
const ORDER_BY = "recent";
const PAGE_SIZE = 10;
```

Reply:
또한, 이 부분의 경우 getArticle에 기본값이 설정되어 있기 때문에 getArticle에서 별도의 기본값을 넣어줄 필요는 없습니다.

Reply:
index에서는 getArticle에서 설정한 기본값 그대로 쓰면 되니까 생략했습니다~!!🫡